agent/src/beerocks/slave/CMakeLists.txt

```diff
@@ -5,22 +5,8 @@ project (${PROJECT_NAME})
 message("${BoldWhite}Preparing ${BoldGreen}${PROJECT_NAME}${BoldWhite} for the ${BoldGreen}${TARGET_PLATFORM}${BoldWhite} platform${ColourReset}")
 
 # Use BPL package from mAP Framework
-find_package(bcl REQUIRED)
-find_package(bwl REQUIRED)
 find_package(btl REQUIRED)
 
-##############################################
-# There is an unresolved depenency of BPL in MapfCommon
-# The dependency was not resolved due to BPL is to be replaced by Platform Service
-# Both those find_package should be deleted as one.
-find_package(MapfCommon REQUIRED)
-find_package(bpl REQUIRED)
```

Comment:
slave should be dependent on BPL with other platforms. 
If you'll ever want to sync back this repo, it will break the buid...

Reply:
you're probably are right - we will fix it then.